test/provider_os_memory.cpp

```diff
@@ -295,3 +297,48 @@ TEST_P(umfProviderTest, purge_force_INVALID_POINTER) {
     verify_last_native_error(provider.get(),
                              UMF_OS_RESULT_ERROR_PURGE_FORCE_FAILED);
 }
+
+// Tests for allocations on numa nodes
+TEST_F(test, checkNUMAnodesAllocation) {
```

Comment:
That's because `bind()` fails with errno=22 (Invalid argument):
```
Breakpoint 1, os_mbind (addr=0x7ffff7ffa000, len=4096, mode=2, nodemask=0x555555625900, maxnode=2, flags=0) at /home/ldorau/work/unified-memory-framework/src/provider/provider_os_memory_linux.c:110
110         return mbind(addr, len, mode, nodemask, maxnode, flags);
(gdb) p *nodemask
$1 = 2
```

Reply:
Done. It should stop failing now.

Reply:
It is OK now